lib/web_ui/test/golden_tests/engine/canvas_arc_scuba_test.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Let's not pub "scuba" in the names of new tests. Let's use the word "golden" instead. I'm also planning to rename the existing tests.

Reply:
Fair, I'll rename them in the next rev (alongside adding the latest hash for goldens)

Reply:
_(Renamed everything under engine, with git mv)_

Reply:
:+1: